src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -246,6 +246,8 @@ public function getTestsForParse()
             array('{\'foo\': \'bar\', "bar": \'foo: bar\'}', array('foo' => 'bar', 'bar' => 'foo: bar')),
             array('{\'foo\'\'\': \'bar\', "bar\"": \'foo: bar\'}', array('foo\'' => 'bar', 'bar"' => 'foo: bar')),
             array('{\'foo: \': \'bar\', "bar: ": \'foo: bar\'}', array('foo: ' => 'bar', 'bar: ' => 'foo: bar')),
+            array('{foo:bar: "baz"}', array('foo:bar' => 'baz')),
```

Comment:
The current deprecation is in part wrong, values such as `{foo:[bar]}` or `{foo:}` are valid, but it is triggered.
What about replacing it by `The behaviour of colon not followed by an indicator (e.g " ", ",", "{", "}", "[", "]") is deprecated since 3.2. It will be considered as part of the value in 4.0.`?